### PR TITLE
Some general UI touches

### PR DESCRIPTION
**Pull request by RubenRBS**
One thing led to another and ended up changing lots of smaller things.
Besides the UI stuff, there's a bit of code cleanup.

The idea is that you _don't_ notice any of the changes individually if you're not looking for them,
but it leaves a better impression overall.

As a PR so someone can check if that's the case, or if I broke something unexpectedly

### CODE REVIEW

File: static/explorer.scss [hunk 1]
```diff
@@ -347,7 +344,7 @@ pre.content.wrap * {
 
 .compiler-list {
     display: block;
-    overflow: auto;
+    height: auto;
```
Comment (partouf) [inline static/explorer.scss:347]: have you tested this? this doesn't work for long lists with limited space 
image

**Review by partouf — CHANGES_REQUESTED** [review]
First glance; just 1 thing I'm not sure of. Mostly looks good, haven't tested though.

File: static/explorer.scss (hunk 1 above)
Comment (RubenRBS) [reply static/explorer.scss:347]: I did, but not this exact configuration. Grat catch! :)

Comment (RubenRBS) [reply static/explorer.scss]: Fixed

**Pull request merged**